template_scripts/add_3rd_party_software.sh

```diff
@@ -86,34 +86,37 @@ if [ ${VERSION} -ge 20 ]; then
     #### '----------------------------------------------------------------------
     info ' Installing rpmfusion-free repos'
     #### '----------------------------------------------------------------------
-    yumInstall /tmp/3rd_party_software/rpmfusion-free-release-${VERSION}.noarch.rpm
-
-    # Disable rpmfusion-free repos
-    yumConfigRepository disable rpmfusion-free > /dev/null
-    yumConfigRepository disable rpmfusion-free-debuginfo > /dev/null
-    yumConfigRepository disable rpmfusion-free-source > /dev/null
-    yumConfigRepository disable rpmfusion-free-updates > /dev/null
-    yumConfigRepository disable rpmfusion-free-updates-debuginfo > /dev/null
-    yumConfigRepository disable rpmfusion-free-updates-source > /dev/null
-    yumConfigRepository disable rpmfusion-free-updates-testing > /dev/null
-    yumConfigRepository disable rpmfusion-free-updates-testing-debuginfo > /dev/null
-    yumConfigRepository disable rpmfusion-free-updates-testing-source > /dev/null
+    if [ -e "/tmp/3rd_party_software/rpmfusion-free-release-${VERSION}.noarch.rpm" ]; then
```

Comment:
Mostly in case of still not released RPM like Fedora 32. So yes better at a message about it.

Reply:
There is rpmfusion for fedora 32 here: https://rpmfusion.org/Configuration

Reply:
It probably arrived few days ago. Anyway, in case of early Fedora Rawhide builds, it does not exist and it's always a need to quote those parts like missing Qubes repository too.

Reply:
In this case, better add a setting to skip rpmfusion. But still, without that setting, fail the build if rpmfusion is missing.

Reply:
The check doesn't work, because the path is related to chroot. See the build log on travis.